src/AWS.Logger.SeriLog/AWS.Logger.SeriLog.csproj

```diff
@@ -30,7 +30,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Serilog" Version="2.5.0" />
+    <PackageReference Include="Serilog" Version="2.10.0" />
```

Comment:
Does the fix require upgrading to `2.10.0` or can we stay on `2.5.0`?

Reply:
@ppittle Customer disn't ask, but there was similar ticket for Log4Net logger, and since we are upgrading `Serilog.Settings.Configuration` to latest version to fix test suite, I thought it would be good to upgrade Serilog reference as well.

Reply:
I think we should take the lowest version dependency possible on Serilog.  That way if a customer is already using Serilog they wont need to worry about upgrading their dependency to use our library.

Reply:
I will revert the change to upgrade the Serilog version to minimize the scope of this PR, test it locally and push new changes.

Reply:
I have min-upgraded `Serilog` to version `2.6.0` (from `2.5.0`) since it was anyway required by old and new (updated). version of `Serilog.Settings.Configuration`.

Reply:
Agree with @ppittle's point where it's seems nice to stay at minimum if possible. I believe the Log4Net upgrade initially started due to a CVE for the version we were on. 

Could the test project depend on 2.6.0 if it's required there, then the source stays at 2.5.0?

Reply:
I would prefer to keep test project and source at same level, i.e., at `2.6.0` since the package `Serilog.Settings.Configuration` still had the issue in the previous version and the last good version is dependent on `Serilog 2.6.0`.